test/module/irohad/model/providers/model_crypto_provider_test.cpp

```diff
@@ -31,6 +38,18 @@ iroha::model::Transaction create_transaction() {
   return tx;
 }
 
+iroha::model::Block create_block() {
```

Comment:
It would be nice to have some kind of shared test_utils library that can be used to share sample blocks and transactions for use in various tests that we right.